client/me/next-steps/index.jsx

```diff
@@ -130,91 +101,84 @@ const NextSteps = createReactClass( {
 				</div>
 			);
 		}
-	},
+	}
 
-	newestSite: function() {
-		return sortBy( this.props.sites.get(), property( 'ID' ) ).pop();
-	},
+	renderOutroMessage() {
+		const { isWelcome, newestSite, translate } = this.props;
 
-	outroMessage: function() {
-		var site, dismissLink;
-
-		if ( this.props.isWelcome ) {
-			site = this.newestSite();
-			dismissLink = '/stats/insights/' + ( site ? site.slug : '' );
+		if ( isWelcome ) {
+			const dismissLink = '/stats/insights/' + ( newestSite ? newestSite.slug : '' );
 
 			return (
 				<div className="next-steps__outro">
 					<p>
-						{ this.props.translate(
+						{ translate(
 							'If you want you can {{a}}skip these steps{{/a}}. You can come back to this page any time.',
 							{
 								components: {
-									a: <a href={ dismissLink } onClick={ this.dismissLinkRecordEvent } />,
+									a: <a href={ dismissLink } onClick={ this.recordDismissLinkClick } />,
 								},
 							}
 						) }
 					</p>
 				</div>
 			);
 		}
-	},
-
-	userHasPurchasedAPlan: function() {
-		return this.props.sites.get().some( function( site ) {
-			return productsValues.isPlan( site.plan );
-		} );
-	},
+	}
 
-	renderSteps: function() {
-		var site = this.newestSite(),
-			sequence = steps.defaultSequence;
+	renderSteps() {
+		const { hasPlan, isWelcome, newestSite } = this.props;
+		let sequence = steps.defaultSequence;
 
-		if ( this.userHasPurchasedAPlan() ) {
+		if ( hasPlan ) {
 			sequence = steps.hasPlanSequence;
 		}
 
 		return (
 			<div className="next-steps__steps">
-				{ sequence.map(
-					function( stepName, index ) {
-						var step = steps.definitions( site )[ stepName ];
-						return (
-							<NextStepsBox
-								key={ stepName }
-								stepName={ stepName }
-								step={ step }
-								primary={ index === 0 }
-								isWelcome={ this.props.isWelcome }
-							/>
-						);
-					}.bind( this )
-				) }
+				{ sequence.map( ( stepName, index ) => {
+					const step = steps.definitions( newestSite )[ stepName ];
+
+					return (
+						<NextStepsBox
+							key={ stepName }
+							stepName={ stepName }
+							step={ step }
+							primary={ index === 0 }
+							isWelcome={ isWelcome }
+						/>
+					);
+				} ) }
 			</div>
 		);
-	},
-
-	render: function() {
-		var classes = 'main main-column next-steps';
+	}
 
-		this.setUserStateFromTrophiesData();
-
-		if ( this.props.isWelcome ) {
-			classes += ' is-single-page';
-		}
+	render() {
+		const classes = classNames( 'main', 'main-column', 'next-steps', {
+			'is-single-page': this.props.isWelcome,
```

Comment:
This _appears_ to be an attempt to render a full-width container with no sidebar. It also _appears_ to have no affect, there are not other `is-single-page` in `assets` or `client`.

Want to rip it out?

Also, we could try to do this _right_ with something like the following:

```
dispatch( setSection( null, { hasSidebar: false } ) );
```

see https://github.com/Automattic/wp-calypso/pull/18796#discussion_r144772226

Probably best to handle this in follow-up, as this PRs already a bit on the large side 🙂

Reply:
Nice catch, and I love the idea you're suggesting here ❤️ .

I've removed that class and removed `classNames` usage as it's no longer necessary: 179be311ca7719319ccc0ddfbeec2c9fa63beead.

Also, I've applied your suggestion for removing the sidebar. To be honest, I was planning to do it in another PR, but since it's super quick and easy, I did it here. Note that it introduces a change to the `/me/next/welcome` page - it'll now be properly centered. Thanks again for suggesting that.